config/environments/development.rb

```diff
@@ -35,6 +35,15 @@
   # Checks for improperly declared sprockets dependencies.
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
+  Paperclip.options[:command_path] = "/usr/local/bin/"
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      bucket: 'sichacvah',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.